src/Compilers/Core/MSBuildTask/InteractiveCompiler.cs

```diff
@@ -204,16 +204,11 @@ protected override string GenerateCommandLineCommands()
         /// </summary>
         protected override string GenerateFullPathToTool()
         {
-            string pathToTool = ToolLocationHelper.GetPathToBuildToolsFile(ToolName, ToolLocationHelper.CurrentToolsVersion);
+            var pathToTool = Utilities.GenerateFullPathToTool(ToolName);
```

Comment:
@agocke all code was refactored to `Utilities.GenerateFullPathToTool()`. Having a new base type for both of them just to have one call to`Log.LogErrorWithCodeFromResources()` seems excessive to me. Do you disagree?